### PR TITLE
refactor: uniform md style & grammarly

### PR DESCRIPTION
**Pull request by JonasKruckenberg**
- remove usage of `Alert`
- remove usage of `title: <document title>` in favor of `# <document title>`
- Use Reference style links
- Apply grammarly suggestions 😄

### CODE REVIEW

File: docs/distribution/updater.md [hunk 1]
```diff
@@ -172,9 +173,9 @@ listen("tauri://update-status", function (res) {
 
 Your server should determine whether an update is required based on the [Update Request](#update-requests) your client issues.
 
-If an update is required your server should respond with a status code of [200 OK](http://tools.ietf.org/html/rfc2616#section-10.2.1) and include the [update JSON](#update-server-json-format) in the body. To save redundantly downloading the same version multiple times your server must not inform the client to update.
+If an update is required, your server should respond with a status code of [200 OK] and include the [update JSON](#update-server-json-format) in the body.
 
-If no update is required your server must respond with a status code of [204 No Content](http://tools.ietf.org/html/rfc2616#section-10.2.5).
+If no update is required, your server must respond with [204 No Content] status code.
```
Comment (FabianLars) [inline docs/distribution/updater.md]: that one sounds weird imo

File: docs/distribution/macos.md [hunk 2]
```diff
@@ -1,43 +1,60 @@
...
+# macOS Application Distribution
+
+Tauri applications for macOS are distributed either with an [Application Bundle] (`.app` files) or an Apple Disk Image (`.dmg` files). The Tauri CLI automatically bundles your application code in these formats, providing options to codesign and notarize your application.
 
 ## Binary targets
 
-You can compile your application targeting Apple Silicon, Intel-based Mac computers or universal macOS binaries. By default, the CLI builds a binary targetting your machine's architecture, but if you are using an Apple Silicon macOS you can compile Intel and universal binaries using the `target` option:
+You can compile your application targeting Apple Silicon, Intel-based Mac computers, or universal macOS binaries. By default, the CLI builds a binary targetting your machine's architecture. If you are using an Apple Silicon macO,S you can compile Intel and universal binaries using the `target` option:
```
Comment (FabianLars) [inline docs/distribution/macos.md]: macO,S typo

**Review by FabianLars — CHANGES_REQUESTED** [review]

File: docs/distribution/updater.md (hunk 1 above)
Comment (JonasKruckenberg) [reply docs/distribution/updater.md]: Agree, I reverted it

File: docs/distribution/macos.md (hunk 2 above)
Comment (JonasKruckenberg) [reply docs/distribution/macos.md]: corrected

**Comment by nothingismagick** [comment]
Please don't apply US English spelling rules for grammarly.

image

**Comment by JonasKruckenberg** [comment]
> Please don't apply US English spelling rules for grammarly.
> 
> image

But it should be consistent, so we choose british?

**Comment by amrbashir** [comment]
it is worth mentioning that throughout tao and tauri we are using `set_maximized` and `set_minimized` ...etc

**Comment by nothingismagick** [comment]
oh that's true @amrbashir - ok, overruled. I take back my comment

**Comment by FabianLars** [comment]
Do you want to run this on the windows distribution guide too or is this pr good to go? I'd like to get it out of the way :P

**Comment by JonasKruckenberg** [comment]
@FabianLars it's good to go

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (FabianLars) [inline .vscode/settings.json]: we generally don't commit ide/editor configs, do we make an exception for this one? I'd be fine with it in this specific case.

Comment (JonasKruckenberg) [reply .vscode/settings.json]: thaaat was actually not on purpose 😅 I guess we remove it, that config won't help anyone whoe doesn't have a paid grammarly subscription + vscode extension soooo

Comment (JonasKruckenberg) [reply .vscode/settings.json]: I removed it

**Review by FabianLars — APPROVED** [review]

**Comment by JonasKruckenberg** [comment]
I'll add the style guidelines as one last thing before we merge

**Comment by JonasKruckenberg** [comment]
@FabianLars sign-off from me, I moved the guidelines to a separate CONTRIBUTING.md. Good to merge 🚀

File: README.md [hunk 4]
```diff
@@ -15,64 +15,6 @@ The **API docs** are generated from our [Rust](https://github.com/tauri-apps/tau
 $ yarn
```
Comment (FabianLars) [inline README.md]: Kinda weird that this single command is still here ngl (well moving the command in general is kinda weird but alright). We should add a link from the readme to the contributing guide tho, for the less github savvy people.

Comment (JonasKruckenberg) [reply README.md]: fixed!

**Pull request merged**